### PR TITLE
docs: add manage roles doc

### PR DESCRIPTION
**Pull request by CharLotteiu**
Signed-off-by: Charlotte Liu <37295236+CharLotteiu@users.noreply.github.com>
This PR is translated from: https://github.com/chaos-mesh/website-zh/pull/23

- Add manage role doc
- Add related pictures

### CODE REVIEW

**Comment by CharLotteiu** [comment]
@WangXiangUSTC PTAL. Thanks~

File: docs/manage-roles.md [hunk 1]
```diff
@@ -1,6 +1,150 @@
...
+
+After you click the link, another window pops up as follows:
+
+![Dashboard Token Generator](img/token_helper.png)
+
+The steps to create user accounts and bind permissions are as follows. You need to perform the first three of the following steps in the pop-up window:
+
+1. Choose the permission scope
+
+   If you want to give the account the appropriate permissions for all Chaos experiments in Kubernetes, check the **Cluster scoped** box. If you specify a namespace in the**Namespace**dropdown option box, the account only has permissions in that specified namespace.
```
Comment (WangXiangUSTC) [inline docs/manage-roles.md:30]:
```suggestion
   If you want to give the account the appropriate permissions for all Chaos experiments in Kubernetes, check the **Cluster scoped** box. If you specify a namespace in the **Namespace** dropdown option box, the account only has permissions in that specified namespace.
```

File: docs/manage-roles.md [hunk 2]
```diff
@@ -1,6 +1,150 @@
...
+
+If you need to manage tokens, click the **Settings** button shown in the left side bar on the Dashboard web page:
+
+![Dashboard Token Management](img/token_manager.png)
+
+In the right part of this page, you can add new tokens in the **Add token** window, or click the **Use** button at the bottom of the page to switch between tokens with different permissions, or click the square button to delete a token.
+
+### Enable or disable permission authentication
+
+If Chaos Mesh is installed using Helm, the permission authentication feature is enabled by default.For production environments and other scenarios with high security requirements, it is recommended to keep the permission authentication feature enabled.If you just want to give Chaos Mesh a try and quickly create Chaos experiments with the permission authentication feature disabled, you can set `--set dashboard.securityMode=false`  in a Helm command. The command is as follows:
```
Comment (WangXiangUSTC) [inline docs/manage-roles.md:143]:
```suggestion
If Chaos Mesh is installed using Helm, the permission authentication feature is enabled by default.For production environments and other scenarios with high security requirements, it is recommended to keep the permission authentication feature enabled.If you just want to give Chaos Mesh a try and quickly create Chaos experiments with the permission authentication feature disabled, you can set `--set dashboard.securityMode=false` in a Helm command. The command is as follows:
```

**Review by WangXiangUSTC — APPROVED** [review]
rest LGTM

File: docs/manage-roles.md (hunk 1 above)
Comment (CharLotteiu) [reply docs/manage-roles.md]: Updated in https://github.com/chaos-mesh/website/pull/80/commits/271e72b544b62b96b9a1119d1c6261a6fa313473.

File: docs/manage-roles.md (hunk 2 above)
Comment (CharLotteiu) [reply docs/manage-roles.md]: Updated in https://github.com/chaos-mesh/website/pull/80/commits/271e72b544b62b96b9a1119d1c6261a6fa313473.

**Comment by CharLotteiu** [comment]
@WangXiangUSTC PTAL again~

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@WangXiangUSTC Please remove the `do-not-merge` label otherwise the PR cannot be merged.

**Pull request merged**